worldedit-core/src/test/java/com/sk89q/worldedit/internal/expression/ExpressionTest.java

```diff
@@ -181,13 +171,11 @@ public void testQuery() throws Exception {
     }
 
     @Test
-    public void testTimeout() throws Exception {
-        try {
-            simpleEval("for(i=0;i<256;i++){for(j=0;j<256;j++){for(k=0;k<256;k++){for(l=0;l<256;l++){ln(pi)}}}}");
-            fail("Loop was not stopped.");
-        } catch (EvaluationException e) {
-            assertTrue(e.getMessage().contains("Calculations exceeded time limit"));
-        }
+    public void testTimeout() {
+        EvaluationException e = assertThrows(EvaluationException.class,
+            () -> simpleEval("for(i=0;i<256;i++){for(j=0;j<256;j++){for(k=0;k<256;k++){for(l=0;l<256;l++){ln(pi)}}}}"),
+            "Loop was not stopped.");
+        assertTrue(e.getMessage().contains("Calculations exceeded time limit"));
```

Comment:
Not your change, but just noticed this should probably be changed to ExpressionTimeoutException instead of depending on the message.

Reply:
Narrowed it, but I still like asserting on the message, to ensure we don't get a different timeout of some sort.